### PR TITLE
Resolve headers already sent issue.

### PR DESCRIPTION
**Pull request by balavec**
Resolve headers already sent issue.
Add .idea folder used in PhpStorm to gitignore.

### CODE REVIEW

**Comment by balavec** [comment]
It would seem I need some help:
Why is Travis not getting proper configuration: Braintree\Configuration::merchantId needs to be set?

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 composer.phar
 .env
 /vendor/
+/.idea/
```
Comment (crookedneighbor) [inline .gitignore]: This should be part of your global gitignore. No need to include it in this project. Please remove this line.

**Comment by crookedneighbor** [comment]
Regarding your question about the Travis build: there are some hidden environmental variables set in the Travis control panel. They are only used when PRs from within the repo are opened. PRs opened from forks will fail, because they can't access the env variables.

Don't worry about the build, we'll run the tests manually before anything gets merged.

**Comment by crookedneighbor** [comment]
@balavec Any thoughts on the feedback/questions I provided?

**Comment by crookedneighbor** [comment]
I'm gonna close this, since we haven't heard back. Feel free to comment here if you'd like to continue the discussion.

**Pull request closed without merging**